### PR TITLE
Minor updates to installable PWA codelab

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR makes sure changes made in https://github.com/GoogleChrome/web.dev/pull/4622 are reflected in the associated codelab article.

Note that the codelab was also updated:
- I've asked Glitch support folks to restore it as it was [broken/empty previously](https://github.com/GoogleChrome/web.dev/issues/4632).
- A new service worker offline page logic was added to address [upcoming changes](https://developer.chrome.com/blog/improved-pwa-offline-detection/).
- Links have been fixed.

@petele Can you review/merge?

### CODE REVIEW

File: src/site/content/en/progressive-web-apps/codelab-make-installable/index.md [hunk 1]
```diff
@@ -25,8 +25,8 @@ It also has an install button that is hidden by default.
 
 When the browser fires the `beforeinstallprompt` event, that's the indication
 that the Progressive Web App can be installed and an install button can be shown
-to the user. The `beforeinstallprompt` event is fired when the PWA meets the
-criteria for installability.
+to the user. The `beforeinstallprompt` event is fired when the PWA meets [the
+criteria for installability](/install-criteria/).
```
Comment (jpmedley) [inline src/site/content/en/progressive-web-apps/codelab-make-installable/index.md:29]:
```suggestion
installability criteria](/install-criteria/).
```

File: src/site/content/en/progressive-web-apps/codelab-make-installable/index.md [hunk 2]
```diff
@@ -107,5 +108,5 @@ Congratulations, you app is now installable!
 
 Here are some additional things that you can do:
 
-+  [Detecting if your app is launched from the home screen](https://developers.google.com/web/fundamentals/app-install-banners/#detect-mode)
++  [Detecting if your app is launched from the home screen](/customize-install/#detect-mode)
 +  [How to show the native app install prompt instead](https://developers.google.com/web/fundamentals/app-install-banners/native)
```
Comment (jpmedley) [inline src/site/content/en/progressive-web-apps/codelab-make-installable/index.md:112]:
```suggestion
+  [Detect if your app is launched from the home screen](/customize-install/#detect-mode)
+  [Show the native app install prompt instead](https://developers.google.com/web/fundamentals/app-install-banners/native)
```

Comment (jpmedley) [reply src/site/content/en/progressive-web-apps/codelab-make-installable/index.md:112]: The best practice for lists is to use parallel grammatical structures.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/progressive-web-apps/codelab-make-installable/index.md (hunk 1 above)
Comment (beaufortfrancois) [reply src/site/content/en/progressive-web-apps/codelab-make-installable/index.md:29]: Fixed.

File: src/site/content/en/progressive-web-apps/codelab-make-installable/index.md (hunk 2 above)
Comment (beaufortfrancois) [reply src/site/content/en/progressive-web-apps/codelab-make-installable/index.md:112]: Fixed.

**Comment by beaufortfrancois** [comment]
@jpmedley I've addressed your feedback. Thanks!

**Review by jpmedley — APPROVED** [review]

**Pull request merged**